service/src/main/java/com/ge/predix/acs/service/policy/evaluation/PolicyEvaluationServiceImpl.java

```diff
@@ -203,6 +204,18 @@ private PolicyEvaluationResult evalPolicySet(final PolicySet policySet, final St
             }
             result = new PolicyEvaluationResult(effect, new ArrayList<>(subjectAttributes),
                     new ArrayList<>(resourceAttributes), resolvedResourceUris);
+        } catch (QueryException qe) {
+            result = new PolicyEvaluationResult(Effect.INDETERMINATE);
```

Comment:
Please move this code block into its own method. This method is very long already (needs to be refacored in a separate story)